test/scales/linearScaleTests.ts

```diff
@@ -188,9 +187,9 @@ describe("Scales", () => {
     });
 
     describe("autoranging behavior", () => {
-      var data: any[];
-      var dataset: Plottable.Dataset;
-      var scale: Plottable.Scales.Linear;
+      let data: any[];
+      let dataset: Plottable.Dataset;
+      let scale: Plottable.Scales.Linear;
```

Comment:
If we let a variable be uninitialized to a value, that's essentially a definition that `const` should not be used to that variable.